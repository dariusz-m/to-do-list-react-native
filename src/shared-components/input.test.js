import renderer from 'react-test-renderer';
import React from 'react';
import Input from './input';

it('Input component renders correctly', () => {
    const tree = renderer.create(
        <Input autoCorrect={true} label={'Label'} onChange={jest.fn()} placeholder={'Placeholder...'} value={'value'}/>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
