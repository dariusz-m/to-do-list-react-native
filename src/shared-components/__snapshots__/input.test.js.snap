// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
    Label
  </Text>
  <TextInput
    allowFontScaling={true}
    autoCorrect={true}
    onChangeText={[MockFunction]}
    placeholder="Placeholder..."
    secureTextEntry={false}
    style={
      Object {
        "borderColor": "black",
        "borderWidth": 0.2,
        "height": 40,
      }
    }
    underlineColorAndroid="transparent"
    value="value"
  />
</View>
`;
